src/components/usageReport/SendMoreCTA.js

```diff
@@ -61,11 +47,8 @@ export class SendMoreCTA extends Component {
             {/* on a deprecated plan */}
             <Case condition={onPlanWithStatus('deprecated')} children={<PageLink to="/account/billing">Switch to a new plan.</PageLink>} />
 
-            {/* is self serve billing and doesn't have online support */}
-            <Case condition={all(isSelfServeBilling, not(hasOnlineSupport))} children={<PageLink to="/account/billing">Upgrade your account.</PageLink>} />
-
-            {/* has online support and is active account status */}
-            <Case condition={all(hasOnlineSupport, hasStatus('active'))} children={this.renderSupportTicketCTA()} />
+            {/* regardless of self serve billing */}
+            <Case children={<PageLink to="/account/billing">Upgrade your account.</PageLink>} />
```

Comment:
This case is not showing up at all.
I think this needs a condition, maybe: 
```suggestion
            <Case condition={() => true} children={<PageLink to="/account/billing">Upgrade your account.</PageLink>} />
```